test/modm/math/filter/moving_average_test.cpp

```diff
@@ -117,3 +131,17 @@ MovingAverageTest::testFloatAverage()
 		TEST_ASSERT_EQUALS_DELTA(filter.getValue(), dataF[i].output, double(1e-4));
 	}
 }
+
+void
+MovingAverageTest::testFloatReset()
+{
+	modm::filter::MovingAverage<TestDataFloat::Type, 4> filter;
+
+	for (uint_fast8_t i = 0; i < 4; ++i)
+	{
+		filter.update(dataF[i].input);
+	}
+
+	filter.reset(42.23);
+	TEST_ASSERT_EQUALS(filter.getValue(), 42.23);
```

Comment:
```suggestion
	TEST_ASSERT_EQUALS_DELTA(filter.getValue(), 42.23, double(1e-4));
```
to make sure this works with lower resolution 32-bit floats too, rather than the 80-bit doubles or whatever the x86 CI uses.

Reply:
When to use `TEST_ASSERT_EQUALS_FLOAT` rather than `TEST_ASSERT_EQUALS_DELTA`?

Reply:
Oh, in this case it should probably be `TEST_ASSERT_EQUALS_FLOAT`, I just copied the line from the check above it, since I forgot the details of the unit test checks again…

Reply:
The `TEST_ASSERT_EQUALS_FLOAT` has a fixed epsilon, while with `TEST_ASSERT_EQUALS_DELTA` the epsilon can be specified in case your computations are more numerically unstable and yield higher differences.

Reply:
Got it. Thanks for your always rich explains!

Reply:
TEST_ASSERT_EQUALS_DELTA is kinda bandpass.